internal/translator/translator_test.go

```diff
@@ -50,4 +50,28 @@ func TestTranslation(t *testing.T) {
 			t.Fatalf("Expected %s, but got %s", "translated el", translation.Phrase)
```

Comment:
The old tests failed. Did you try the tests locally?

![image](https://user-images.githubusercontent.com/1281019/85572725-79e53900-b5f2-11ea-89b6-43f301e15e8f.png)

Reply:
The issue is in the filter part.

Reply:
It was because the first rule didn't have Type so the filter returned the last rule in the array because it didn't have with what type compare and I had a problem with an array of test rules.
But now it's solved.